src/CourseAuthoringRoutes.test.jsx

```diff
@@ -75,54 +63,51 @@ describe('<CourseAuthoringRoutes>', () => {
 
   fit('renders the PagesAndResources component when the pages and resources route is active', () => {
```

Comment:
```suggestion
  it('renders the PagesAndResources component when the pages and resources route is active', async () => {
```
Because of the `fit` instead of `it`, some of the tests in this file are not running, and they are broken. Also, this test isn't really working because the `waitFor` needs to be `async`

Reply:
Nice catch! Corrected